Synview-backend/websocket/websocket_route.ts

```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from "../generated/prisma/client.ts";
+import { Session } from "https://deno.land/x/oak_sessions/mod.ts";
+import { withAccelerate } from "@prisma/extension-accelerate";
+import AuthMiddleware from "../middleware/auth_middleware.ts";
+import { Router } from "@oak/oak";
+import { EntrySocket } from "./websocket_server.ts";
+
+const env = Deno.env.toObject();
+type AppState = {
+  session: Session;
+};
+
+const wsRouter = new Router<AppState>();
+
+wsRouter.use(AuthMiddleware);
```

Comment:
Would recommend a good test of the AuthMiddleware to make sure it is working properly

1. Shut down the server and reopen it. Previously issued JWT tokens should still work
2. Make up a fake/wrong JWT token and make sure it fails the auth check
3. Set a super short expiration time on the token and make sure it fails